nn/conformer.py

```diff
@@ -0,0 +1,204 @@
+"""
+Conformer code.
+Ref: https://arxiv.org/abs/2005.08100
+"""
+
+from typing import Tuple, List, Union
+from . import Module, ModuleList, LayerRef, Linear, dropout, layer_norm, batch_norm, Conv, swish, glu, split_dims, \
+    merge_dims, pool
+
+
+class _PositionwiseFeedForward(Module):
+    """
```

Comment:
Indentation should be consistent (2 spaces).

Reply:
ah yes. I will configure my PyCharm settings to be consistent.